scripts/menu-script/13-createcustomfile.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+while true; do
+    echo
+    read -p "-> Is this a resource or data source? [r|d] " rd
+    case $rd in
+            [Rr]* ) 
+            dir1="resources"; break;;
+
+            [Dd]* ) 
+            dir1="data-sources"; break;;
+            * ) echo; echo "Please answer resource or data source [r|d]. ";; 
+    esac
+done
+echo
+read -p "Please enter your resource or datasource name. " resourcename
+path=../../templates/$dir1
```

Comment:
I would suggest surrounding variables used for path and file manipulation with curly braces to avoid unexpected bash-isms.

```suggestion
path=../../templates/${dir1}
```